src/pages/Staking/index.tsx

```diff
@@ -1,10 +1,723 @@
 
+import * as React from 'react';
```

Comment:
Yes, this file is huge. But the major big part is not JSX but the calculation. So this is big because of the escrow API structure to some extent. I follow this paradigm https://epicreact.dev/one-react-mistake-thats-slowing-you-down/ most of the time.

If you have a good idea, please let me know when we have a call. Thanks.